packages/frontend/src/pages/campaigns/index.tsx

```diff
@@ -38,6 +38,11 @@ const STATE_OPTIONS = [
     },
 ];
 
+export interface OrderOption {
```

Comment:
Not sure why we need this? Can't we use the `SelectOption` exported by the `Select` component in the UI lib it should be the exact same data structure and the name difference doesn't increase readability that much.

Reply:
it was complaining because the value can be a string on SelectOption. I guess I can cast it somewhere and use the select option,

Reply:
Ohhh yeah, we can use the casting. Admittedly, we can improve typings on the `Select`

Reply:
fixed: a26cf15c061aa03f79ceb3d7f5e489fdc74ef5c7